.github/dependabot.yml

```diff
@@ -0,0 +1,35 @@
+version: 2
```

Comment:
Out of curiosity, why are you updating weekly instead of daily, which is the default iirc?

Reply:
We went for weekly and grouping dependencies together because before that the commit tree was basically only dependabot commits.

Reply:
Is that a problem? By going for weekly updates fixes such as potential security issues will take up to 1 week to land and, also, grouping dependencies will make harder to pinpoint issues if one breaks the build, or even the deployments. The smaller the delta between releases, the lower the risk

Reply:
In some cases we had the same dependency update twice or three times during the week. for example from `x.y.1` to `x.y.2` and then to `x.y.3`. And I think dependabot doesn't stick to this schedule when it is addressing a security alert.
But we will re-evaluate this strategy, because I agree with you that sometimes when the CI breaks it isn't at all straightforward to figure out which dependency caused the failure.